### PR TITLE
New VEP_plugins debian package with new plugin and a plugin fix

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -3,3 +3,10 @@ vepplugins-1 (1) unstable; urgency=low
   * Initial Release.
 
  -- The Genome Institute <gmt@genome.wustl.edu>  Mon, 29 Oct 2012 15:00:00 -0600
+
+vepplugins-2 (2) unstable; urgency=low
```
Comment (apregier) [inline debian/changelog]: Usually the latest entry goes at the top of the file (I think)

**Comment by apregier** [comment]
+1 (I assume you were able to package and install it?)

File: modules/Wildtype.pm [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        WildtypeProtein     => "The normal, non-mutated protein sequence",
+    };
+}
+
+sub run {
+    my ($self, $tva) = @_;
+
+    my $tv = $tva->transcript_variation;
+    my $tr = $tv->transcript;
+    my $cds_seq = defined($tr->{_variation_effect_feature_cache}) ? $tr->{_variation_effect_feature_cache}->{translateable_seq} : $tr->translateable_seq;
```
Comment (acoffman) [inline modules/Wildtype.pm]: It seems a little odd that we check this condition two different places. Additionally it looks like the fallback value of `1` for `$codon_table` exists in several places. Could we do something like this instead:

``` perl
my $cds_seq;
my $codon_table;
if (defined($tr->{_variation_effect_feature_cache}) {
  $cds_seq = $tr->{_variation_effect_feature_cache}->{translateable_seq};
  $codon_table = $tr->{_variation_effect_feature_cache}->{codon_table};
} else {
  $cds_seq =  $tr->translateable_seq;
  my ($attrib) = @{$tr->slice->get_all_Attributes('codon_table')};
  if ($attrib) {
    $codon_table = $attrib->value;
  }
}

$codon_table ||= 1;
```

**Comment by acoffman** [comment]
+1 either way!

File: modules/Wildtype.pm (hunk 2 above)
Comment (susannasiebert) [inline modules/Wildtype.pm]: I like this suggestion but I would probably preset `$codon_table` to 1 and then overwrite as needed. Or is there a drawback to that approach?

**Pull request merged**